sound/soc/codecs/rt715-sdca.h

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: GPL-2.0 
+// 
+// rt715-sdca.h -- RT715 ALSA SoC audio driver header
+// 
+// Copyright(c) 2019 Realtek Semiconductor Corp.
+// 
+
+#ifndef __RT715_SDCA_H__
+#define __RT715_SDCA_H__
+
+#include <linux/regulator/consumer.h>
```

Comment:
you include only one header and it's the one you don't need :-) Please, include all needed headers (e.g. workqueue.h, sdw.h) and forward-declare all structs, that aren't defined in those headers or here.